Formula/cockroach.rb

```diff
@@ -19,10 +19,6 @@ class Cockroach < Formula
   depends_on "xz" => :build
 
   def install
-    # unpin the Go version
-    go_version = Formula["go"].installed_version.to_s.split(".")[0, 2].join(".")
-    inreplace "src/github.com/cockroachdb/cockroach/.go-version",
-              /^GOVERS = go.*/, "GOVERS = go#{go_version.gsub(".", "\\.")}.*"
```

Comment:
FYI Homebrew maintainers: we've removed the Go version pinning upstream, largely for the convenience of package managers like yourselves. :)

Reply:
Thanks!